Makefile

```diff
@@ -60,10 +61,14 @@ XSMM_DIR ?= ../libxsmm
 OCCA_DIR ?= ../occa
 
 # env variable MAGMA_DIR can be used too
-MAGMA_DIR ?= ../magma
+#TODO: FIX THIS with hip detection...
+#MAGMA_DIR ?= ../magma
 # If CUDA_DIR is not set, check for nvcc, or resort to /usr/local/cuda
-CUDA_DIR  ?= $(or $(patsubst %/,%,$(dir $(patsubst %/,%,$(dir \
+#CUDA_DIR  ?= $(or $(patsubst %/,%,$(dir $(patsubst %/,%,$(dir \
```

Comment:
With the above change, I can comment this line out and all backends work fine.

Reply:
Okay, seems like we can leave CUDA and MAGMA as-is for now, and assume that the machine I'm using is a particularly weird setup? I reverted all these lines back to what they were before.